Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -98,8 +142,15 @@ public class ConnectionPool {
         give(connection)
     }
     
-    /// Release all the connections in the pool by calling connectionReleaser closure on each connection,
-    /// and empty the pool.
+    // MARK: Disconnect Pool
+    /**
+     Release all the connections in the pool by calling `connectionReleaser` closure on each connection, and empty the pool.
+     ### Usage Example: ###
+     A `previously initialized `ConnectionPool` is disconnected removing all connections in the pool.
```

Comment:
The backtick needs to be removed from before previously so it reads -
"A previously initialized `ConnectionPool` is disconnected, thereby removing all connections from the pool."